webapp.js

```diff
@@ -49,12 +48,7 @@ app.use(express.static('dist'));
 app.use(bodyParser.json());
 
 app.get('/api/data', function(req, res) {
```

Comment:
Oh read it too quickly.. `db` gets set asynchronously in that callback.. I guess the way you could make it work is to declare all the routes only after `db` is defined. Not sure what the convention with express is.. probably can just leave then.